Gruntfile.js

```diff
@@ -181,5 +169,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-browserify');
 
   grunt.registerTask('default', ['exec:bower_update','clean','copy:fonts','sass:dev','browserify:dev','exec:build_sphinx','connect','open','watch']);
-  grunt.registerTask('build', ['exec:bower_update','clean','copy:fonts','sass:build','browserify:build','uglify','exec:build_sphinx']);
+  grunt.registerTask('build',   ['clean','copy:fonts','sass:build','browserify:build','uglify']);
+  grunt.registerTask('install', ['exec:bower_update','clean','copy:fonts','sass:build','browserify:build','uglify']);
+  grunt.registerTask('docs',    ['exec:build_sphinx','connect','open']);
```

Comment:
This should also call all the stuff that copies and builds assets (`copy`, `sass`, `browserify`). The docs need these assets for proper display and they might not be there if the were `clean`ed.